ee/tabby-schema/Cargo.toml

```diff
@@ -27,6 +27,7 @@ tracing.workspace = true
 validator = { version = "0.16.1", features = ["derive"] }
 regex.workspace = true
 hash-ids.workspace = true
+tabby-git = { path = "../../crates/tabby-git" }
```

Comment:
what's this for?

Reply:
Need to be able to read metadata (refs) from the repositories we're returning